tests/console/install_single_package.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Simple test for installing any given package
+# Maintainer: soulofdestiny <mgriessmeier@suse.com>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    select_console 'root-console';
+
+    pkcon_quit;
+
+    # add specific repository which contains the package
+    if (get_var('PACKAGE_REPO')) {
```

Comment:
but why PACKAGE_REPO but PACKAGETOINSTALL? sounds very inconsistent to me.

And why a copyright of 2009? Because of the 3 use lines?